lib/omniauth/flexible_strategy.rb

```diff
@@ -72,6 +72,14 @@ def path_for_provider(name)
       "#{path_prefix}/#{name}"
     end
 
+    ##
+    # This method returning false does not mean that the current request should be handled by
+    # this strategy. The method can, however, indicate that a request should NOT be handled by it.
+    # In which case it returns true.
+    def not_on_auth_path?
+      (current_path =~ /#{path_prefix}/) != 0
+    end
```

Comment:
i think this is equivalent to `current_path !~ /\A#{path_prefix}/` but maybe more readable?

> "current path starts not with path prefix"